include/roboteam_utils/networking/src/Channels.cpp

```diff
@@ -10,7 +10,7 @@ const std::map<ChannelType, Channel> CHANNELS = {
     {ROBOT_COMMANDS_PRIMARY_CHANNEL, {"commands_primary", "127.0.0.1", "5559"}},
     {ROBOT_COMMANDS_SECONDARY_CHANNEL, {"commands_secondary", "127.0.0.1", "5560"}},
     {FEEDBACK_PRIMARY_CHANNEL, {"feedback_primary", "127.0.0.1", "5561"}},
-    {FEEDBACK_SECONDARY_CHANNEL, {"feedback_yellow", "127.0.0.1", "5562"}},
+    {FEEDBACK_SECONDARY_CHANNEL, {"feedback_secondary", "127.0.0.1", "5562"}},
```

Comment:
Why was this changed?

Reply:
To be consistent with the PRIMARY SECONDARY notation, and because it could be confusing, as the channel could (depending on referee settings), also contain feedback from blue robots.